test/integration/test_bind.rb

```diff
@@ -10,7 +10,9 @@ def test_bind_timeout
     error = assert_raise Net::LDAP::Error do
       @ldap.bind(method: :simple, username: "uid=user1,ou=People,dc=rubyldap,dc=com", password: "passworD1")
     end
-    assert_equal('Connection timed out - user specified timeout', error.message)
+    msgs = ['Operation timed out - user specified timeout',
+            'Connection timed out - user specified timeout']
+    assert_send([msgs, :include?, error.message])
```

Comment:
This exposes the implementation too much, would it be possible to have separate tests for the two different types of errors?